src/Client/Stripe.php

```diff
@@ -73,4 +80,12 @@ public function createSetupIntent(StripeCustomerId $stripeCustomerId): SetupInte
     public function retrievePaymentMethod(StripeCustomerId $customerId, StripePaymentMethodId $methodId): PaymentMethod;
 
     public function detatchPaymentMethod(StripePaymentMethodId $paymentMethodId): void;
+
+    public function searchCustomers(array $searchParams): SearchResult|Collection;
+
+    /**
+     * Returns a list of PaymentMethods for Treasury flows.
+     * @return \Stripe\Collection<\Stripe\PaymentMethod>
+     */
+    public function listAllPaymentMethodsForTreasury(array $array): Collection;
```

Comment:
As discussed, we'll probably phase out this call and use 'list for customer' per recommendation at https://docs.stripe.com/api/payment_methods/list